db/models/order.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+const {STRING, TEXT, FLOAT, JSON, ENUM, DATE, INTEGER, BOOLEAN} = require('sequelize')
+const states = [
+  'AL',
+  'AK',
+  'AZ',
+	'AR',
+	'CA',
+	'CO',
+	'CT',
+	'DE',
+	'FL',
+	'GA',
+	'HI',
+	'ID',
+	'IL',
+	'IN',
+	'IA',
+	'KS',
+	'KY',
+	'LA',
+	'ME',
+	'MD',
+	'MA',
+	'MI',
+	'MN',
+	'MS',
+	'MO',
+	'MT',
+	'NE',
+	'NV',
+  'NH',
+	'NJ',
+	'NM',
+	'NY',
+	'NC',
+	'ND',
+	'OH',
+	'OK',
+	'OR',
+	'PA',
+	'RI',
+	'SC',
+	'SD',
+	'TN',
+	'TX',
+	'UT',
+	'VT',
+	'VA',
+	'WA',
+	'WV',
+	'WI',
+	'WY']
+
+module.exports = db => db.define('order', {
+  products: {
+    type: JSON,
+  },
+  subtotal: {
+    type: FLOAT,
+    defaultValue: 0.00
+  },
+  tax: {
+    type: FLOAT,
+    defaultValue: 0.00
+  },
+  address: {
+    type: STRING,
+  },
+  city: {
+    type: STRING,
+  },
+  state: {
+    type: ENUM(...states),
+  },
+  zip: {
+    type: STRING,
+    allowNull: true,
+    validate: {
+      len: [5]
+    }
+  },
+  shippingMethod: ENUM('Express', 'Standard'),
+  isPurchase: {
+    type: BOOLEAN,
+    defaultValue: false
+  },
```

Comment:
Date we be when it's created, not actually when the purchase is made. Will probably have to update that as well once the front end request is finally made to purchase.